centrallix/objectsystem/obj_datatypes.c

```diff
@@ -1769,9 +1769,9 @@ objDataCompare(int data_type_1, void* data_ptr_1, int data_type_2, void* data_pt
 			    }
```

Comment:
Hmm, it looks like the old implementation expects to be given 2 integers, so it can compare the first integer to WholePart and the second integer to FractionPart. With the new MoneyType representation, it might make more sense to only expect one integer to compare to, but I'm not sure - I think we should ask Greg about it when he gets back. Maybe leave a comment about this so we remember to ask about that?

Reply:
Scratch this, I just realized this is for comparing an _IntVec_ type to a MoneyType, not an Integer type 🤪